.github/workflows/test.yml

```diff
@@ -0,0 +1,13 @@
+on: [push]
```

Comment:
It looks like the purpose of a typical `test.yml` should be running automated tests for the project. Please change it according to the jtest project, although we don't have any tests now.